projects/plugins/crm/tests/php/automation/class-automation-workflow-test.php

```diff
@@ -312,6 +314,7 @@ public function test_workflow_execution_with_condition_false() {
 		$automation = new Automation_Engine();
 		$automation->set_automation_logger( $logger );
 		$automation->register_trigger( Contact_Created_Trigger::class );
+		$automation->register_step( Contact_Condition::class );
```

Comment:
Our hardcoded dummy workflow data had hardcoded the class name for the mock "Contact Condition" step, so we never had to register the step.
This was technically a false positive I had to correct after I updated the dummy data, but is otherwise unrelated to the PR itself.